### DISCUSSION TITLE
What is the minimum amount of storage for Runpod's Network Volume needed to install and run this worker (excluding models)

### DISCUSSION THREAD

**Discussion by Valachio** [ashleykleynhans/runpod-worker-a1111#16] [Q&A] [answered]
Assuming you follow [the official instructions to the letter](https://github.com/ashleykleynhans/runpod-worker-a1111/blob/main/docs/installing.md) and install all the supported extensions and excluding the models (SDXL & Deliberate v2)

Excluding any models, how many GB of storage on Runpod's Network Volume is needed to successfully complete setting up the worker?

It'll be nice to know a minimum amount to reduce cost. You can increase Network Volume disk size anytime, but cannot reduce it.

**Comment by ashleykleynhans** [comment, marked as answer]
I don't know, I use around 50GB to install everything including the models. Best to try it out for yourself.

**Comment by houmie** [comment]
I have the same issue.  I started with 10 GB, ran out of space. Then tried 20 GB, no luck., third time tried 40 GB no luck... was about to write here to ask. So you really think 50GB is enough? It would be nice to mention it here: https://github.com/ashleykleynhans/runpod-worker-a1111/blob/main/docs/installing.md

**Comment by ashleykleynhans** [comment]
there is no minimum, it all depends on your own personal requirements, mostly about how many models you want to use, so I'm not adding it to the docs because its unpredictable.

**Comment by houmie** [comment]
Thanks yeah 50gb worked. The issue is every time you run the script it deletes everything and starts from scratch. :-) 

Sorry, I have another question, In step 3, I should have downloaded more models that I would be interested in right? Something like `dreamshaper-xl10` correct?  I think I can do it later too, because I deleted the pod now.

**Reply by ashleykleynhans** [reply]
Yes you can do it later, just attach a pod to your storage and download models manually.
